docs/Deploying-Pega-on-PKS.md

```diff
@@ -194,6 +195,54 @@ Use the provided example addons.yaml file to configure the use of a the Traefik
   - To use the default EFK settings, you must set a domain name to access kibana from your load balancer using the `kibana.hosts: "YOUR_WEB.KIBANA.EXAMPLE.COM"` parameter.
 
   - If your TKGI deployment already has log aggregation capabilities configured, you must disable EFK deploy by setting the `deploy_efk: &deploy_efk false` parameter.
+  
+  
+### Updating the backingservices.yaml Helm chart values (Supported when installing or upgrading to Pega Infinity 8.6 and later)
+  
+  Adding the Pega configuration files to your Helm installation on your local system
+  ----------------------------------------------------------------------------------
+  Pega maintains a repository of Helm charts that are required to deploy Pega Platform and related resources using Helm, including a generic version of the following charts. After you add the repository to your local system, you can customize these configuration files for your backing service deployment:
+  
+  - pega/backingservices - Use this chart to set customization parameters for your deployment. You will modify this chart later in the deployment tasks.
+  
+  1. To add the backingservices repository to your Helm installation, enter:
+  
+      `$ helm repo add pega https://dl.bintray.com/pegasystems/pega-helm-charts`
```

Comment:
Do you want this to be updated to latest repo instead of bintray `helm repo add pega https://pegasystems.github.io/pega-helm-chart`s as per latest [README ](https://github.com/pegasystems/pega-helm-charts/blob/master/README.md#getting-started) I see this in couple of other README's in the same PR.